packages/flutter/test/widgets/scrollbar_test.dart

```diff
@@ -53,9 +53,12 @@ void main() {
   final _DrawRectOnceCanvas testCanvas = _DrawRectOnceCanvas();
   ScrollbarPainter painter;
 
-  Rect captureRect() => testCanvas.rects.removeLast();
+  Rect captureRect() => testCanvas.rects.single();
```

Comment:
I believe single is a property and not a method.

Reply:
darn my attempts to edit via github UI